calico/felix/felix.py

```diff
@@ -48,7 +48,7 @@
 from calico.felix.fetcd import EtcdAPI
 
 _log = logging.getLogger(__name__)
-
+_log.setLevel(logging.DEBUG)
```

Comment:
I'm confused by the rearrangement of newlines in this file:

* There should be two newlines about the definition of `_main_greenlet`, as there were before.
* It's not clear why you've added the newlines below